<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElephAnt</class>
 <widget class="QMainWindow" name="ElephAnt">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ElephAnt - Trunkloads of Power for Antenna Measurements</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,0,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,6">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <attribute name="headerMinimumSectionSize">
           <number>256</number>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>256</number>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Setup List</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>New Setup</string>
           </property>
           <property name="statusTip">
            <string extracomment="Click here to create a new setup"/>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_general">
           <attribute name="title">
            <string>General Information</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2,2">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Setup Information</string>
                </property>
                <layout class="QFormLayout" name="formLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Setup Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="tb_setup_name"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Comment</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="tb_setup_comment"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Last Modified</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="l_last_modified">
                   <property name="text">
                    <string>---</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>AUT Information</string>
                </property>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>AUT Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="tb_aut_name"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_hwSetup">
           <attribute name="title">
            <string>Hardware Setup</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Log Output:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="logDisplayArea">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Setup"/>
    <addaction name="actionOpen_Setup"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Setup"/>
    <addaction name="actionSave_Setup_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Setup"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Setup">
   <property name="text">
    <string>Open Setup</string>
   </property>
  </action>
  <action name="actionNew_Setup">
   <property name="text">
    <string>New Setup</string>
   </property>
  </action>
  <action name="actionSave_Setup">
   <property name="text">
    <string>Save Setup</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClose_Setup">
   <property name="text">
    <string>Close Setup</string>
   </property>
  </action>
  <action name="actionSave_Setup_As">
   <property name="text">
    <string>Save Setup As ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
